packages/core-utils/src/time.ts

```diff
@@ -103,8 +107,12 @@ export function getLongDateFormat(config) {
  * @param {number} seconds duration in seconds
  * @returns {string} formatted text representation
  */
-export function formatDuration(seconds) {
-  return formatDurationLikeMoment(seconds, false);
+// TS TODO: region as type?
+export function formatDuration(seconds: number, region: string): string {
```

Comment:
We should deprecate this method (implementers now use formatjs)

Reply:
Deprecated in db1658fad93479daa48f7229371b9aef2200ae86